spec/routes/web/spec_helper.rb

```diff
@@ -39,6 +39,24 @@ def refresh
     Capybara.reset_sessions!
     Capybara.use_default_driver
   end
+
+  RSpec::Matchers.define :have_flash_message do |expected_type, expected_message|
```

Comment:
Can we have two matchers here, `have_flash_notice` and `have_flash_error`?  On failure, can we show the content of both the flash notice and error.  I think that will make debugging much easier.

Reply:
I refactored it. Also update failure message

Example failure message:

     Failure/Error: expect(page).to have_flash_notice("An email has been sent to you with a link to verify your account2")

             expected: notice - An email has been sent to you with a link to verify your account2
         actual error: (no error message)
        actual notice: An email has been sent to you with a link to verify your account